### DISCUSSION TITLE
How do I restrict deletion of duplicates to only those that span across directories?

### DISCUSSION THREAD

**Discussion by avekifes** [qarmin/czkawka#1232] [Q&A]
Okay, say I have scanned two folders...

X:\windows\mac
X:\windows\windows

...and the results comprise a mixture of duplicate groups where:

1. there are duplicates in only "X:\windows\mac"
2. there are duplicates in only "X:\windows\windows"
3. there are duplicates that span across "X:\windows\mac" and "X:\windows\windows"

Is there a way I could *only select* the duplicates that belong to the third category, and *only those* that are in "X:\windows\mac," for deletion?

Example: "linux" is a duplicate file that exists in both "X:\windows\mac" and "X:\windows\windows"
I *only want* to delete the duplicate versions of "linux" in "X:\windows\mac" but keep the ones in "X:\windows\windows"

**Comment by qarmin** [comment]
"Reference directories" are directories that are treated as originals, so cannot  be selected and only visible are records, that have duplicates in different folders - this needs to be selected before scan(checkbox should be visible in "Included Directories" tab)

**Reply by avekifes** [reply]
Need a bit of clarification on how this works. Let's say I have the "mac" folder set as a reference folder:

X:\windows\mac (reference)
X:\windows\windows

Does that mean czkawka/krokiet will only show duplicates that are found **_in both_** "X:\windows\mac" and X:\windows\windows"? Or will it also show duplicates that are only in "X:\windows\windows"?

What I'm trying to figure out is if it's safe to select/delete everything that shows up as a duplicate using these settings. Or do I run the risk of accidentally deleting all versions (original **_and_** duplicates) of a file?

**Reply by qarmin** [reply]
If there are 4 duplicated files:
- X:\aaa
- X:\bbb
- Y:\ccc
- Y:\ddd

then adding X:\ and Y:\ to scan, will find and group all 4 together
If X will be set as reference folder, then only one group will be visible

- X:\aaa - select button will be not clickable
- Y:\ccc
- Y:\ddd

so in this situation "Select All" will select only Y:\ccc and Y:\ddd

![Screenshot from 2024-03-05 21-29-59](https://github.com/qarmin/czkawka/assets/41945903/99034d59-a7d7-477a-b892-3a10448b5119)
![Screenshot from 2024-03-05 21-30-10](https://github.com/qarmin/czkawka/assets/41945903/3faa6252-b5fb-4d1e-bd81-65408f20a22e)

**Comment by PJDude** [comment]
Hi, what You need is a "Cross paths mode" available in [DUDE](https://github.com/PJDude/dude)
